docs/outlook/delegate-access.md

```diff
@@ -15,11 +15,14 @@ The following table shows supported client-server combinations for this feature,
 
 | Client | Exchange Online | Exchange 2019 on-premises<br>(Cumulative Update 1 or later) | Exchange 2016 on-premises<br>(Cumulative Update 6 or later) | Exchange 2013 on-premises |
 |---|:---:|:---:|:---:|:---:|
-|Windows:<br>version 1910 (build 12130.20272) or later|Yes|No|No|No|
+|Windows:<br>version 1910 (build 12130.20272) or later|Yes|In preview<sup>1</sup>|In preview<sup>1</sup>|In preview<sup>1</sup>|
 |Mac:<br>build 16.47 or later|Yes|Yes|Yes|Yes|
 |Web browser:<br>modern Outlook UI|Yes|Not applicable|Not applicable|Not applicable|
 |Web browser:<br>classic Outlook UI|Not applicable|No|No|No|
 
+> [!NOTE]
+> <sup>1</sup> Support for this feature in an on-premises Exchange environment is currently in preview starting from version 2205 (build 15228.10000).
```

Comment:
Since it's a single item, not sure if we should number it...Maybe just using an asterisk or something like that for now
```suggestion
> \* Support for this feature in an on-premises Exchange environment is currently in preview starting from version 2205 (build 15228.10000).
```